Assets/FeatureVoid/PlatformMemorizer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlatformMemorizer : MonoBehaviour {
+
+    // This script must be attached to the mecha
+    // in order to keep track of the last platform
+    // it landed on
+    // This feature is used to reposition the mecha
+    // after falling into the void
+
+    private GameObject lastPlatForm;
+
+	// Use this for initialization
+	void Start () {
+        // Use a random platform for initialization
+        lastPlatForm = GameObject.FindGameObjectWithTag("Platform");
```

Comment:
That's clearly not a good idea. We'd maybe better ensure that the mecha spawns on a correct platform, and register it

Reply:
The Last Platform can now be manually initialized.